crates/chia-consensus/src/gen/owned_conditions.rs

```diff
@@ -14,7 +14,7 @@ use chia_py_streamable_macro::{PyJsonDict, PyStreamable};
     pyo3::pyclass(name = "Spend", get_all, frozen),
```

Comment:
you don't think the python name should be changed as well?

Reply:
Done, with the caveat that the chia-blockchain change is gonna have to land first

Reply:
you could also export it under both names

Reply:
in fact, I think it would be problematic to bump the `chia-blockchain` dependency. The version we use right now was carefully chosen to still contain python implementations of the classes we have ported to rust, so we can test against them